src/components/StaticAppBar/StaticAppBar.css

```diff
@@ -19,6 +19,9 @@
 .drawerAppBar button{
     margin-top: 0px !important;
 }
+.top-menu{
```

Comment:
@isuruAb Please don't change this has been fixed already in https://github.com/fossasia/chat.susi.ai/pull/687